modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -379,6 +381,12 @@ public void testIncompleteRebalanceAfterResetPartitions() throws Exception {
 
         assertRealAssignments(node0, partId, 0, 1, 3);
 
+        Assignments assignment013 = Assignments.of(timestamp,
```

Comment:
What is the reason for moving this piece of code?

Reply:
@Cyrill asked me because for him it looks more understandable. I tend to agree with him as he is the person who is not in the context and he faced some troubles with understanding tests

Reply:
Ok